src/es/components/molecules/tab/Tab.js

```diff
@@ -0,0 +1,122 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+export default class Tab extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+
+    const selectors = {
+      button: 'button',
+      buttonActive: 'button.active',
+      contentActive: 'div[tab-content-target]#'
+    }
+
+    const attributes = {
+      tabTarget: 'tab-target'
+    };
+
+    const states = {
+      active: 'active',
+      show: 'show'
+    };
+
+    // Show content of default active tab
+    this.tabs = this.root.querySelectorAll(selectors.button);
```

Comment:
this could also be accomplished with a getter [get tab () {return this.root.query...}]